### PR TITLE
Clarify deploying a virtual cluster

### PR DESCRIPTION
**Pull request by ritawatson**
This PR:
- Carifyies 'Deploying a Palette Virtual Cluster' 
- Revises the entire doc. 
- Adds tabs to gain real estate in the doc so it's not one long doc. 
- Changes the doc title to "Add Virtual Clusters to a Host Cluster" (previously titled "Quickstart with Palette Virtual Clusters").

Jira: https://spectrocloud.atlassian.net/browse/DOC-422

[Preview](https://deploy-preview-1015--docs-spectrocloud.netlify.app/clusters/palette-virtual-clusters/virtual-cluster-quickstart)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1015#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (amarjeet27) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:121]: Let's invert the actions - select a project then click Clusters from the menu.

The sequence doesn't matter if the user is already in the context of a project, but it matters if the user is a Tenant Admin. If a Tenant Admin clicks Clusters, then selected a project they see the Project Overview page, and have to click Clusters once again.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (amarjeet27) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:124]: Complete the **Deploy New Virtual Cluster**...
deploy

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:130]: Notes should use the InfoBox component.  We also have large whitespaces between these elements that need to be reduced.

CleanShot 2022-12-02 at 07 37 24

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:145]: Let's always point to the next step in the journey. A good next suggestion is to encourage the user to deploy an application by creating an app profile. Let's point to the `/devx/app-profile/create-app-profile` create an app profile page.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:149]: Important information like this belongs at the top. People skim so let's help them by raising this higher up in the document.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:121]: Great suggestion Amarjeet 😄

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback provided.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:130]: I wondered about that. I'll change it.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:121]: That makes sense - I'll switch the actions.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:124]: I thought these labels stay the same because we discussed when the Palette logo doesn't display at top left to give context, the UI will keep the word 'Palette' in labels. Did I misunderstand?

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:149]: I'm not sure I understand what to move. Should **Deploying a Palette Virtual Cluster** be moved before **Enabling Palette Virtual Clusters on an Existing Host Cluster**?

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:145]: Added: 
Next, try deploying an application by [creating an app profile](/devx/app-profile/create-app-profile).

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:149]: @ritawatson the infoBox, it contains important gotcha information.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:124]: @amarjeet27 I see the label now that you point out and removed 'Palette' in the label name.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:171]: @ritawatson  We need to remove this InfoBox as it's duplicated now on the page.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:161]: We shouldn't bold the word must.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1015#pullrequestreview-1203068482" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1015#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Rita and I discussed the need to conduct some restructuring.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1015#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (tiwarisumit3) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:29]: Clusters with the Palette Virtual Cluster feature enabled or added as part of Cluster Group are referred to as **Host Clusters**.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (tiwarisumit3) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:74]: @ritawatson - looks like this page was wrote probably when we only supported direct deployment on host clusters. I think deploying via the Cluster Group option can also be part of the same document.

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1015#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1015#pullrequestreview-1212361857" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — APPROVED** [review]
Minor changes that need some attention but otherwise it's looking good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1015#pullrequestreview-1213720224" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1015#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1015#pullrequestreview-1212361857" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
